lib/spack/spack/schema/config.py

```diff
@@ -107,7 +107,7 @@
                 "names": ["install_missing_compilers"],
                 "message": "The config:install_missing_compilers option has been deprecated in "
                 "Spack v0.23, and is currently ignored. It will be removed from config in "
-                "Spack v0.25.",
+                "Spack v1.0.",
```

Comment:
Can we reword to "after Spack v1.0", instead of "in Spack v1.0"? That would give us a bit more flexibility (expected removal was Nov. 2025 at the time of writing this warning)

Reply:
Good idea. I'll get that changed.